.travis.yml

```diff
@@ -14,17 +14,15 @@ addons:
   apt:
     packages:
       - rpm
+  chrome: stable
```

Comment:
@mikeller it's there. This was technically invalid yaml because there were 2 `addons` keys on the same level, so duplicate keys in dictionary pretty much.

Reply:
:+1: